src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -267,7 +236,7 @@ GetNativeCalendarName
 
 Gets the native calendar name.
 */
-CalendarDataResult
+ResultCode
```

Comment:
Nit: is this being on its own line just how the code formatter is choosing to do it due to line length?  Seems odd that it's on its own line here but not in the previously exited section.